external/os-autoinst-common/.gitrepo

```diff
@@ -6,7 +6,7 @@
 [subrepo]
 	remote = git@github.com:os-autoinst/os-autoinst-common.git
 	branch = master
-	commit = 50328b0211a5a170e3ba1013e9f85601211062ad
+	commit = e555e723f0e784d1973b5f141a5ec4a432a71b9e
```

Comment:
I don't find that commit in os-autoinst master. Did you run that command from the master branch?

Reply:
oh sorry. that's the commit from os-autoinst-common. The one below `parent` is the commit in os-autoinst. I can find that one 9b31e8770b86dfbb479e325f494d0b7cdbf56d66 just wondering why it is such an old commit. But I guess it will be ok.